front/components/vaults/CreateVaultModal.tsx

```diff
@@ -63,7 +69,10 @@ export function CreateVaultModal({
     disabled: true, // Disable as we just want the mutation function
   });
   const router = useRouter();
-  const { members, isMembersLoading } = useMembers(owner);
+  const { members, isMembersLoading } = useMembers({
+    owner,
+    returnLight: true,
```

Comment:
This name feels weird. Can we simply go with either mode: "light" or something more concise?